### PR TITLE
Standardize on import numpy as np

### PR DESCRIPTION
**Pull request by BabaYaga1221**
- [X] I hereby agree to dual license this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit``
locally, and understand that continuous integration checks will be used to
confirm the Biopython unit tests and style checks pass with these changes.

- [x] I have added my name to the alphabetical contributor's listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, are listed
already or do not wish to be listed. (*This acknowledgment is optional.*)

Closes #...
Update the ``import numpy`` with the ``import numpy as np`` standard.

### CODE REVIEW

File: Bio/Affy/CelFile.py
Comment (peterjc) [inline Bio/Affy/CelFile.py:19]: NumPy is how the project capitalises their name. Don't change this.

**Comment by peterjc** [comment]
Duplicates work already in progress on #4258, will look again once that's merged and this has been black formatted.

Also note this updates a lot of the tests despite discussion on #4238 about concerns for the ``test_Align*.py`` files in particular.

**Comment by BabaYaga1221** [comment]
Ok, Sure. Take your time sir.

**Comment by BabaYaga1221** [comment]
Okay, @peterjc I will inform you as soon as possible.

**Comment by BabaYaga1221** [comment]
@peterjc revert all the changes on ``Test/test_Align*.py`` files.

File: Bio/PDB/vectors.py
Comment (peterjc) [inline Bio/PDB/vectors.py:481]: NumPy for the docstring, not np

File: Bio/PDB/vectors.py
Comment (peterjc) [inline Bio/PDB/vectors.py:508]: NumPy for the docstring, not np

File: Tests/test_MarkovModel.py
Comment (peterjc) [inline Tests/test_MarkovModel.py:24]: I don't think you needed to change any of these imports - maybe just revert all the changes to this file?

File: Bio/LogisticRegression.py
Comment (peterjc) [inline Bio/LogisticRegression.py:22]: I think you can remove this comment.

(I'm assuming this still works - there probably was once a reason to explicitly import linalg)

File: Bio/SVDSuperimposer/__init__.py
Comment (peterjc) [inline Bio/SVDSuperimposer/__init__.py:17]: This was not on the list of things we suggested changing. I would be inclined not to touch this since you've left it with inconsistent import styles.

**Comment by peterjc** [comment]
I really should have been more explicit in writing #4238. Even with most of the test changes removed this is far to long for easy review. I've noted some points though.

**Comment by BabaYaga1221** [comment]
Okay Sir, @peterjc, I will note your work and update the status as soon as possible.

**Comment by peterjc** [comment]
It looks like you accidentally checked in lots of files under ``Tests/__pycache__/`` and then deleted them again?

**Comment by BabaYaga1221** [comment]
It was a mistake, I accidentally created some unnecessary files. That's why, I delete them all in next commit

File: Bio/Cluster/__init__.py
Comment (peterjc) [inline Bio/Cluster/__init__.py:283]: If changing numpy in the text, use NumPy not Numpy.

(4 cases in this file, lines 191, 193, 281 and 283)

**Review by peterjc — APPROVED** [review]
Long, but by eye all the changes look good and I see no formatting issues now (after dropped changes to most of the test files especially, those currently with black-exempt regions).

**Comment by peterjc** [comment]
Note when merging - this should be a squash-and-merge because the history has some mis-steps.

**Comment by peterjc** [comment]
Oh, an issue:

```
Bio/MarkovModel.py:39:11: F821 undefined name 'numpy'
```

**Review by peterjc — CHANGES_REQUESTED** [review]
Please fix ``Bio/MarkovModel.py:39:11: F821 undefined name 'numpy'``

**Comment by BabaYaga1221** [comment]
@peterjc okay sir, i will resolve issue and commit again.

**Comment by BabaYaga1221** [comment]
@peterjc Sir, would you like to review the latest commit?

**Review by peterjc — APPROVED** [review]

**Review by peterjc — COMMENTED** [review]
As noted earlier, much longer (more changes) than I would have liked to review in one pull request - but this looks good now to squash-and-merge.

Given the length, a second review would be sensible. @mdehoon?

**Review by peterjc — APPROVED** [review]

**Review by mdehoon — APPROVED** [review]

**Comment by peterjc** [comment]
Thank you!

**Comment by BabaYaga1221** [comment]
@peterjc Pleasure to work on this project, Sir. In near future, I will try to contribute more enhancement in biopython.

**Pull request merged**
